deploy/openshift/070-ValidatingWebhookConfiguration.yaml

```diff
@@ -21,7 +21,7 @@ webhooks:
         path: /drainer
         port: 443
       caBundle: Cg==
-    admissionReviewVersions: ["v1"]
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
Same as earlier ... `v1beta1` is not supported anymore in Kube 1.22. It is also not handled by the code. So we cannot have it in.

Reply:
Same as above.